packages/app/src/domain/international/select-countries/country-code.ts

```diff
@@ -0,0 +1,68 @@
+export const countryCodes: CountryCode[] = [
+  'aut',
+  'bel',
+  'bgr',
+  'che',
+  'cyp',
+  'cze',
+  'deu',
+  'dnk',
+  'esp',
+  'est',
+  'fin',
+  'fra',
+  'gbr',
+  'grc',
+  'hrv',
+  'hun',
+  'irl',
+  'isl',
+  'ita',
+  'lie',
+  'ltu',
+  'lux',
+  'lva',
+  'mlt',
+  'nld',
+  'nor',
+  'pol',
+  'prt',
+  'rou',
+  'svk',
+  'svn',
+  'swe',
+];
+
+export type CountryCode =
```

Comment:
```suggestion
export type CountryCode = typeof countryCodes[number];
```
(And type countryCodes to `countryCodes = [....] as const`)